pkg/render/guardian.go

```diff
@@ -125,17 +130,22 @@ func (c *GuardianComponent) Objects() ([]client.Object, []client.Object) {
 		c.service(),
 		secret.CopyToNamespace(GuardianNamespace, c.cfg.TunnelSecret)[0],
 		c.cfg.TrustedCertBundle.ConfigMap(GuardianNamespace),
-		// Add tigera-manager service account for impersonation
-		CreateNamespace(ManagerNamespace, c.cfg.Installation.KubernetesProvider, PSSRestricted),
-		managerServiceAccount(),
-		managerClusterRole(false, true, c.cfg.UsePSP, c.cfg.Installation.KubernetesProvider),
-		managerClusterRoleBinding(),
-		managerClusterWideSettingsGroup(),
-		managerUserSpecificSettingsGroup(),
-		managerClusterWideTigeraLayer(),
-		managerClusterWideDefaultView(),
 	)
 
+	if c.cfg.EnterpriseCRDExists {
+		// Add tigera-manager service account for impersonation
+		objs = append(objs,
+			CreateNamespace(ManagerNamespace, c.cfg.Installation.KubernetesProvider, PSSRestricted),
```

Comment:
We may need the namespace / serviceaccount / and RBAC resources.

Managed clusters need to be able to authorize requests from management cluster components. So if `system:serviceaccount:tigera-manager:tigera-manager` is making requests to the managed cluster API server, we will need to make sure that service account exists and is bound to the correct permissions.